backend/endpoints/reload_documents.py

```
@@ -0,0 +1,284 @@
+            evaluate(child, configuration) for child in condition_dict["children"]
+        )
+    else:
+        return configuration.get(condition_dict["id"], None) == condition_dict["value"]
+
+
+def parse_condition(visibility_condition: dict | None) -> dict | None:
+    """Transforms a visibility condition returned by the getConfiguration endpoint into a condition_dict."""
+    if visibility_condition == None:
+        return None
+    elif visibility_condition["btType"] == NONE_CONDITION:
+        return None
```

Ruff (0.12.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)